qiskit/pulse/ir/alignments.py

```diff
@@ -0,0 +1,37 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+class Alignment:
```

Comment:
This should be an abstract base class, right?

Reply:
Yes. I didn't want to put too much work into this until we decide how to deal with the two alignments hierarchies.